src/SampleBuf.jl

```diff
@@ -76,7 +76,7 @@ domain(buf::AbstractSampleBuf) = range(0.0, stop=(nframes(buf)-1)/samplerate(buf
 import Base: +, -, *, /
 import Base.broadcast
 
-const ArrayIsh = Union{Array, SubArray, Compat.LinRange, StepRangeLen}
+const ArrayIsh = Union{Array, SubArray, LinRange, StepRangeLen}
```

Comment:
This definition is kind of stupid... All of these things are subtypes of `AbstractArray`. Thus, you can search and replace in all the `src` folder `ArrayIsh -> AbstractArray` and remove this `ArrayIsh` from the code. Not sure why this complexity was introduced.......

Reply:
It’s been a while but my best guess is that I was defining a union that would include most AbstractArrays but not include SampleBufs. Possibly to avoid some kind of infinite loop on method resolution when combining raw arrays and samplebufs.

it’s been a while though so very possible it’s not necessary anymore (or maybe never was)